utils/prepare_test_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit
+
+# Setup virtualenv
+virtualenv venv
+source venv/bin/activate
+
+# Ensure you have the latest version of pip
+pip install --upgrade pip
+
+# Install packs requirements
+pip install -r requirements.txt
+pip install -r requirements-test.txt
+
+# Checkout and install st2 requirements
+git clone https://github.com/StackStorm/st2.git --depth 1 --single-branch --branch v1.5.1 ./st2
```

Comment:
1. Why are we checking out an other branch here than installed via the St2 bootstrap script? 

2. How can we syncronize this (and the bootstrap install) with the St2 version we actually want to use when we're developing/testing? I'm guessing we would like to have this vs the 1.4 branch atm for example.